modules/security/wifi.go

```diff
@@ -66,11 +66,10 @@ func wifiInfo() string {
 }
 
 func wifiNameLinux() string {
-	cmd := exec.Command("nmcli", "-t", "-f", "in-use,ssid", "dev", "wifi")
-	out := utils.ExecuteCommand(cmd)
-	name := utils.FindMatch(`\*:(.+)`, out)
+  cmd := exec.Command("/usr/sbin/iwgetid", "-r")
```

Comment:
I'm not sure why this is showing as indented differently, it looks the same in vim **and** GitHub's editor.

Reply:
Is this utility guaranteed to always be at that absolute path on all systems?

Reply:
I'm not sure yet, but calling it without absolute path failed for me.